### ISSUE TITLE
Ability to upload Translator documents in the Event form

### ISSUE THREAD

**Issue by Redmayspecial** [unfoldingWord-dev/translationDatabaseWeb#228]
## User Story
As a translation manager I would like the ability to upload documents in support of the fields tracking required translator sign-offs, so that the forms are electronically  associated to the person and the event.

## Definition of Done
Research and test the ability upload and store the following electronic signed documents

 - [ ] Statement of Faith
 - [ ] Translation Guidelines

**Comment by Redmayspecial** [comment]
Middle

**Comment by vleong2332** [comment]
Discuss with door43 (@jag3773) and tS (@neutrinog) on where to upload the file.

**Comment by jag3773** [comment]
The request to publish form already does this.  Check with @cdaield or @paltman on that @vleong2332 .

**Comment by Redmayspecial** [comment]
@jag3773 Yes, I remember that being part of the requirements during our tS training over the summer.  I agree that this could be redundant, but we are hearing the desire from the team to take the documents that they are collecting and have collected and store them now rather than wait for the publishing phase.  Can they do that on the tS side at the start of a language project well in advance of  completion/publishing phase?  If so, then maybe it becomes an issue of training, not development.

**Comment by jag3773** [comment]
@Redmayspecial I wasn't mentioning that it was redundant, I was mentioning that there's already a setup and code to do this that @vleong2332 could look at to copy. The files are uploaded to S3.
